src/presentation/controllers/base-controller.ts

```diff
@@ -22,16 +24,19 @@ interface Route {
   path: string
   method: Method
   action: Action
+  validationSchema: JSONSchema7 | undefined
```

Comment:
Suggestion: I always default to using optionals in these cases 😄 What do you think?

```suggestion
  validationSchema?: JSONSchema7
```

Reply:
I'm also a bigger fan of the optional operator.
The reason I went for the `| undefined` in this case was because of this error:
![image](https://github.com/user-attachments/assets/e1a31e61-c433-4a4a-b190-a3e90f75172e)

If you have a nice suggestion for circumventing that, then I'm all for the optional operator :)

Reply:
Oh right... Webstorm 😄 Haha... i still don't have a license for it.

Will check out what I can do to maybe get the same errors in VSCode, or just get a license 🥲